### PR TITLE
Update redis-py

### PR DESCRIPTION
**Pull request by bensteinberg**
https://github.com/redis/redis-py/releases

https://fakeredis.readthedocs.io/en/latest/about/changelog/

### CODE REVIEW

**Comment by rebeccacremona** [comment]
Cool. Imma pull this down and give it a little exercise since our tests intentionally mock redis / run celery in synchronous mode. Will do later this afternoon!

File: perma_web/requirements.txt [hunk 1]
```diff
@@ -237,10 +241,6 @@ decorator==5.1.1 \
     # via
     #   ipdb
     #   ipython
-deprecated==1.2.13 \
-    --hash=sha256:43ac5335da90c31c24ba028af536a91d41d53f9e6901ddb021bcc572ce44e38d \
-    --hash=sha256:64756e3e14c8c5eea9795d93c524551432a0be75629f8f29e67ab8caf076c76d
-    # via redis
```
Comment (rebeccacremona) [inline perma_web/requirements.txt:243]: I very like that this is going away :-)

**Review by rebeccacremona — APPROVED** [review]
Thanks!

I tested a large link batch with `CELERY_TASK_ALWAYS_EAGER = False` to confirm that celery can still talk to redis, and uploaded a new registrar logo with `CACHES["default"] = ...` configured as with prod and saw that old and new logos were served as expected from the django cache, and turned `AXES` and the `AXES_CACHE` and confirmed that it still locks people out as expected.

So! All the spots we use redis seem good!

**Comment by bensteinberg** [comment]
Thank you for checking this!

**Pull request merged**